src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -623,19 +635,16 @@ bool fx_muxer_t::resolve_sdk_dotnet_path(const pal::string_t& own_dir, pal::stri
             }
         }
     }
-    else
-    {
-        trace::verbose(_X("Failed to obtain current working dir"));
-    }
 
     std::vector<pal::string_t> hive_dir;
     pal::string_t local_dir;
     pal::string_t global_dir;
-    bool multilevel_lookup = multilevel_lookup_enabled();
+
+    bool multilevel_lookup = own_dir.empty() ? true : multilevel_lookup_enabled();
```

Comment:
@nguerrera The point I am trying to make is very simple - multi-level lookup is really about enabling lookup via non-own_dir folders. Whether or not it is enabled should not affect your change at all (and vice-versa), which is why I think the above change should be reversed.

Ofcourse, if own_dir is not specified and MLL is disabled, nothing will happen and that is expected. But forcing MLL to happen even when the user specified it to be disabled, when own_dir is empty, is incorrect.